tests/test_chat.py

```diff
@@ -643,6 +678,34 @@ async def make_assertion(*_, **kwargs):
         monkeypatch.setattr(chat.get_bot(), "copy_message", make_assertion)
         assert await chat.copy_message(chat_id="test_copy", message_id=42)
 
+    async def test_instance_method_forward_from(self, monkeypatch, chat):
+        async def make_assertion(*_, **kwargs):
+            chat_id = kwargs["chat_id"] == chat.id
+            message_id = kwargs["message_id"] == 42
+            from_chat_id = kwargs["from_chat_id"] == "test_forward"
+            return from_chat_id and message_id and chat_id
+
+        assert check_shortcut_signature(Chat.forward_from, Bot.forward_message, ["chat_id"], [])
+        assert await check_shortcut_call(chat.forward_from, chat.get_bot(), "forward_message")
+        assert await check_defaults_handling(chat.forward_from, chat.get_bot())
```

Comment:
please also add these to the new tests above

Reply:
@Bibo-Joshi how did these tests not catch the return type difference (see my review), does it not check for that?

Reply:
https://github.com/python-telegram-bot/python-telegram-bot/blob/76bfe8ceff6e7fcf71fcb9d437eb41fb9c8f1d4b/tests/conftest.py#L502-L505

But mypy should indeed have complained 🤔

Reply:
also checked locally and mypy does indeed not pick that up … don't know why. For now I'm okay with you having catched it :D

Reply:
> don't know why

probably because we are ignoring return type in every bot method?

@aditya-yadav-27 reminder to implement the tests above (cause I didn't see it in the new commit)

Reply:
should I add these to new set_photo, set_description, delete_photo, etc tests?

Reply:
yes

Reply:
> probably because we are ignoring return type in every bot method?

no, I don't think that that's the reason. `Bot.forward_message` is annotated to return a `Message` - that should be all that is checked when checking what the type of `await super().forward_message(…)` is.